pkg/urbit/jets/tree.c

```diff
@@ -575,7 +557,6 @@ static u3j_hood _141_pen__ut_ho[] = {
   { "fine",   49086 },
   { "fire",       4 },
   { "fish",    6006 },
```

Comment:
Since jet core identity does an equality check on the hooks, is this OTAable? My model is that if you won't recognize the compiler jets anymore...

Reply:
Didn't realize that.  I've tested disabling the jets on a -L copy of ~wicdev, but not deleting the jets.  I'll do that, and if it doesn't work I'll re-add the jets as pass-throughs, or maybe there's a way to have the hook but no jet implemented.

Reply:
I believe you can delete the jet registrations without deleting the hook registrations.

Reply:
You can definitely have the hook without the jet.

But i also think there's no problem in removing the hook (I think there would be if we were running under hashboard, but we're not). I think the only thing the `u3j_hood*` declarations are used for with fast-hint registration is lazy name->axis lookup when you actually try to resolve a hook. (And I think this because the hook declarations in `tree.c` are pretty stale ...)

@frodwith, can you confirm?

Reply:
Turns out I was wrong and @eglaysher is right -- I obviously misunderstood this part of the system. Please restore these hooks.

Reply:
Hmm, strangely I can run this on ~wicdev and build a pill with no issues.  Regardless, better to leave the hooks in.

Reply:
When you try that, do the existing `+ut` jets still fire?

Reply:
I just talked to @frodwith some more about this, and I now recant my previous recantation. The combination of hashboard and fast-hints is fairly complicated and difficult to reason about it. And our implementation of fast-hints was glued onto hashboard after the fact; this area definitely needs some work in light of @frodwith's more up-to-date designs.

My earlier claim, that the presence/absence of the hooks is irrelevant (except lazily, in cases where they are actually called), is also not quite right. It looks like it's the hooks in your parent core that are used to match a battery to an already registered driver. And registrations are updated by fast hints. So @eglaysher and I were each about half right.

Reply:
So best to just leave in the hooks?

Reply:
Yes. But it doesn't really matter either way -- getting things working in hashboard again would be a significant chunk of work regardless.